modules/network-modifying-operator-install-config.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
```

Comment:
It would be good to give some motivation - that there are some operator-specific parameters that are not covered as part of the Network.config.openshift.io object.

Reply:
For example, if you want to use Multitenant mode, change the MTU, or the vxlan port, you would need to create the operator configuration manually.